src/mscorlib/src/System/Runtime/Serialization/FormatterServices.cs

```diff
@@ -100,7 +100,7 @@ static FormatterServices()
             Contract.Assert((object)type != null, "[GetAllSerializableMembers]type!=null");
 
             if (type.IsInterface) {
-                return new MemberInfo[0];
+                return Array.Empty<MemberInfo>();
```

Comment:
This kind of runtime formatting doesn't exist in .NET Core, and this array is being returned publically, so there's non-zero compat risk.  If the feature did exist in .NET Core, this is one where I think it could warrant a bit more investigation, as I suspect while formatting it's not uncommon to ask for the serializable members of a type and have that type be an interface.  But all else considered, let's revert this.